tests/type_inference/test_functions.py

```diff
@@ -30,12 +30,51 @@ def test_function_def_call_no_args(function_name, return_value):
     """Test type setting in environment of a function call for a function with no parameters."""
     TYPE_CONSTRAINTS.clear_tvars()
     assume(not iskeyword(function_name))
-    program = _parse_to_function(function_name, [], return_value) + "\n" + function_name + "()\n"
+    program = _parse_to_function(function_name, [], repr(return_value)) + "\n" + function_name + "()\n"
     module = cs._parse_text(program)
     # there should be a single Expr node in this program
     expr_node = next(module.nodes_of_class(astroid.Expr))
     assert expr_node.type_constraints.type == type(return_value)
 
 
+@given(cs.valid_identifier(), hs.lists(cs.valid_identifier(), min_size=1))
+def test_function_def_args_simple_return(function_name, arguments):
+    """Test FunctionDef node visitors representing function definitions with params; return one of it's arguments."""
+    assume(not iskeyword(function_name) and function_name not in arguments)
+    # generate every possible function definition program of aforementioned form.
+    for i in range(len(arguments)):
+        return_value = arguments[i]
+        program = _parse_to_function(function_name, arguments, return_value)
+        module = cs._parse_text(program)
+        # get the functionDef node - there is only one in this test case.
+        function_def_node = [n for n in module.nodes_of_class(astroid.FunctionDef)][0]
+        expected_arg_type_vars = [function_def_node.type_environment.lookup_in_env(argument) for argument in arguments]
+        expected_arg_types = [TYPE_CONSTRAINTS.lookup_concrete(type_var) for type_var in expected_arg_type_vars]
+        function_type_var = module.type_environment.lookup_in_env(function_name)
+        function_type = TYPE_CONSTRAINTS.lookup_concrete(function_type_var)
+        actual_arg_types, actual_return_type = TYPE_CONSTRAINTS.types_in_callable(function_type)[0],\
```

Comment:
You don't need to explicitly access `[0]` and `[1]` here; Python will automatically unpack the tuple.